libs/common/src/lib/select-account-kb/select-kb/select-kb.component.ts

```diff
@@ -72,7 +72,7 @@ export class SelectKbComponent implements OnDestroy {
       )
       .subscribe((result) => {
         this.sdk.refreshKbList();
```

Comment:
I could reproduce the zone issue, I guess in some cases the `nuclia.options.zone` is wrong here.